cellrank/tl/estimators/_base_estimator.py

```diff
@@ -555,6 +532,60 @@ def compute_absorption_probabilities(
 
         self._write_absorption_probabilities(time=start, extra_msg=extra_msg)
 
+    def compute_lineage_priming(
+        self, method: str = "kl_divergence", early_cells: Optional[Sequence[str]] = None
+    ) -> pd.Series:
+        """
+        TODO.
+
+        Parameters
+        ----------
+        method
+            TODO. Valid options are:
+
+                - `'kl_divergence'` - TODO.
+                - `'entropy'` - TODO.
+
+        early_cells
+            Cell ids or a mask marking early cells. If `None`, use all cells. Only used when ``method='kl_divergence'``.
+
+        Returns
+        -------
+        The priming degree.
```

Comment:
Inconsistent. Let's please agree on consistent naming: 
- the method is called `compute_lineage_priming`
- the method should return (and write) `lineage_priming`
- it should have modes `kl_divergence` and `entropy`
- accordingly, the corresponding lineage class methods should be `kl_divergence` and `entropy`, not `priming_degree` and `entropy`, I find this confusing.

Reply:
So we can change to: "The method returns the degree of lineage priming."

Reply:
Ok, so I should remove `entropy` from `__overloaded_functions__` in Lineage, is that right?